lib/rubocop/cop/rspec/expect_actual.rb

```diff
@@ -97,7 +97,7 @@ def simple_literal?(node)
 
         def complex_literal?(node)
           COMPLEX_LITERALS.include?(node.type) &&
-            node.each_child_node.all?(&method(:literal?))
+            node.each_child_node.all? { |child_node| literal?(child_node) }
```

Comment:
Please see the other commits. Would you rather prefer this?
```ruby
            node.each_child_node.all? { literal?(_1) }
```

Reply:
I prefer the current code.

```ruby
node.each_child_node.all? { |child_node| literal?(child_node) }
```

Reply:
I believe there is a cop to disallow numbered parameters, if we want to do so.

Reply:
JFYI, `Style/NumberedParameters` cop has an option `EnforcedStyle: disallow`.
https://docs.rubocop.org/rubocop/1.63/cops_style.html#stylenumberedparameters